packages/nouns-webapp/src/components/NounInfoCard/index.tsx

```diff
@@ -19,31 +19,28 @@ import { useDispatch } from 'react-redux';
 
 interface NounInfoCardProps {
   nounId: number;
+  bidHistoryOnClickHandler?: () => void;
 }
 
 const NounInfoCard: React.FC<NounInfoCardProps> = props => {
-  const { nounId } = props;
+  const { nounId, bidHistoryOnClickHandler } = props;
   const history = useHistory();
   const dispatch = useDispatch();
 
   const etherscanBaseURL = buildEtherscanAddressLink(config.addresses.nounsToken);
-  const bidHistoryButtonClickHandler = () => {
-    dispatch(setOnDisplayAuctionNounId(nounId));
-    history.push(`/auction/${nounId}`);
-  };
+  const bidHistoryButtonClickHandler = bidHistoryOnClickHandler
+    ? bidHistoryOnClickHandler
+    : () => {
```

Comment:
in what case would `bidHistoryOnClickHandler` not be passed in? logically it seems that the bid history button should *always* shows bid history?

ref:
```
I think basically this was for backward compatibility with legacy noun profiles
but I think it probably makes sense to just deprecate those
```